src/pages/home/report/ReportActionCompose.js

```diff
@@ -227,17 +238,24 @@ class ReportActionCompose extends React.Component {
     /**
      * Show the ReportActionContextMenu modal popover.
      *
-     * @param {Object} [event] - A press event.
      */
-    showEmojiPicker(event) {
+    showEmojiPicker() {
         this.textInput.blur();
-        this.state.emojiPopoverAnchorPosition = {
-            horizontal: event.nativeEvent.pageX,
-            vertical: event.nativeEvent.pageY,
-        };
         this.setState({isEmojiPickerVisible: true});
```

Comment:
Here we were showing the Picker based on mouse position. As a result, Piker will be shown on different locations based on click position. Let me know if we want this behavior. I removed it as other Menus does not have this behavior keeping consistency.